tests/components/shelly/conftest.py

```diff
@@ -36,10 +36,16 @@
         "mac": MOCK_MAC,
         "hostname": "test-host",
         "type": MODEL_25,
+        "num_inputs": 3,
         "num_outputs": 2,
     },
     "coiot": {"update_period": 15},
     "fw": "20201124-092159/v1.9.0@57ac4ad8",
+    "inputs": [
+        {"btn_type": "momentary"},
+        {"name": "Test input 1", "btn_type": "momentary"},
+        {"name": "Test input 2", "btn_type": "momentary"},
+    ],
```

Comment:
Here is an example from a real device:
```
    {
      "name": "TV LEDs",
      "btn_type": "momentary",
      "btn_reverse": 0
    },
    {
      "name": "TV Spots",
      "btn_type": "momentary",
      "btn_reverse": 0
    },
    {
      "name": null,
      "btn_type": "momentary",
      "btn_reverse": 0
    }
```
The 3rd input is not set on the device UI and report `null` (this will be translated to `None` when we read it)